### PR TITLE
add kraken@7.5.0 w/ git auto-update

### LINKED ISSUE #8010: [Request] Add kraken
**Library name:** kraken
**Git repository url:** https://github.com/cferdinandi/kraken

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t, please help me review this PR, thanks! 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/cferdinandi/kraken
 * Official website (optional, not the repository): https://cferdinandi.github.io/kraken/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 626
   - Count of watchers: 66 
   - Count of forks: 69
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8010, cc cferdinandi

### CODE REVIEW

File: ajax/libs/Kraken/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Kraken",
+  "filename": "css/main.min.css",
```
Comment (pvnr0082t) [inline ajax/libs/Kraken/package.json]: @nolsherry I think it should point to `js/main.min.js`. What do you think?

Comment (pvnr0082t) [inline ajax/libs/Kraken/package.json]: @nolsherry I think it should point to `js/main.min.js`. What do you think?

Comment (nolsherry) [inline ajax/libs/Kraken/package.json]: @pvnr0082t, but I think css/main.min.css is better.

Comment (pvnr0082t) [inline ajax/libs/Kraken/package.json]: @PeterDaveHello Could you give some advise? Or it need to confirm with author?

Comment (PeterDaveHello) [inline ajax/libs/Kraken/package.json]: both fine, personally I prefer to use js.

**Comment by nolsherry** [comment]
@pvnr0082t, I modified `filename` field in to `js/main.min.js`,
please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
LGTM

**Pull request merged**